src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Extensions/HttpResponseMessageTestExtensions.cs

```diff
@@ -14,15 +15,24 @@ public static void AssertOk(this HttpResponseMessage message)
         Assert.Equal(OK, message.StatusCode);
     }
 
-    public static T AssertOk<T>(this HttpResponseMessage message)
+    public static T AssertOk<T>(this HttpResponseMessage message, bool useSystemJson = false)
     {
         Assert.Equal(OK, message.StatusCode);
 
-        var body = message.Content.ReadFromJson<T>();
+        T? body = Deserialize<T>(message, useSystemJson);
```

Comment:
Not sure why we we'd need to change this from `var`

Reply:
changed back :)

Reply:
habit